examples/SpringBootHikariExample/src/main/java/software/amazon/SpringBootHikariExample/ApiController.java

```diff
@@ -0,0 +1,18 @@
+package software.amazon.SpringBootHikariExample;
```

Comment:
Some files are missing copyright notices, do they need them?

Reply:
Good catch and done! 
I wasnt sure about  the `SpringBootHikariExampleApplication` since it was generated by Spring Boot Initializr. But then i realized it would be faster to rewrite the class from scratch than to look for the answer lol